openml-h2o/pom.xml

```diff
@@ -105,6 +110,13 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>4.0.1</version>
```

Comment:
just for the record, this is the version H2O uses right?

Reply:
Nice point. As a matter of fact, I can remove this as it is included in the h2o-jetty-9 module.